vhpidirect/arrays/intvector/csized/makefile

```diff
@@ -0,0 +1,56 @@
+CC=gcc
```

Comment:
`CC?=gcc`, `GHDL?=ghdl`, `CFLAGS+=-fPIC`, `STD?=93`, etc.

Reply:
cant have it all on one line...

Reply:
Unless I am missing a deliminator

Reply:
Oh, you're highlighting the naivete of the plain assignment... Um... yeah, I suppose I could update.... I don't see any upsides though?

Reply:
Yes. Sorry, I was in a hurry.

The main point is that there are no obvious downsides, and it can be useful. Some use cases:

- `CC=g++` or `CC=clang` are valid options. For example, the latter is used to build GHDL on macOS in CI: https://github.com/ghdl/ghdl/blob/master/dist/ci-run.sh#L399
- Tristan uses `GHDL=` very frequently in order to run testsuites with temporal builds of GHDL. This repo is not a test suite, but we might want to make it easier for him and for us to test examples with non-default locations and names of GHDL binaries.
- By the same token, we might want to easily test some examples with various versions of the standard. Some are for 93 only, others for 08 only, but most of them should work with any version.